utils/src/main/scala/it/pagopa/interop/commons/utils/JWTPathGenerator.scala

```diff
@@ -0,0 +1,17 @@
+package it.pagopa.interop.commons.utils
+
+import it.pagopa.interop.commons.utils.service.OffsetDateTimeSupplier
+
+import java.time.format.DateTimeFormatter
+import java.util.UUID
+
+object JWTPathGenerator {
+  def generateJWTPathInfo(dateTimeSupplier: OffsetDateTimeSupplier): (String, String) = {
```

Comment:
returning a tuple is often confusing because the caller is not able to understand from the signature the obtained value.
I suggest to create and return a very simple case class with both values